### ISSUE TITLE
builds.release.core-os.net started returning 403s

### ISSUE THREAD

**Issue by mumoshu** [coreos/bugs#2207]
# Issue Report #

### Container Linux Version ###

Any

### Environment ###

AWS

### Expected Behavior ###

http://builds.release.core-os.net is accessible from anonymous clients so that we can avoid building every packages in order to build/install NVIDIA drivers.

### Actual Behavior ###

Not sure exactly when, but at some point it started returning 403 like

https://gist.github.com/kylegato/689a366b62f744702b634e710cd4cc76

### Reproduction Steps ###

  1. Provision a kube-aws cluster with NVIDIA GPU enabled.

### Other Information ###

https://github.com/kubernetes-incubator/kube-aws/pull/985#discussion_r145674270

**Comment by euank** [comment]
Oops, we messed up!

These devfiles are normally uploaded to `gs://builds.developer.core-os.net`, which is anonymously accessible and listable.
The devfiles for the version you're using (1520.6.0) are there (e.g. http://builds.developer.core-os.net/boards/amd64-usr/1520.6.0/pkgs/sys-kernel/coreos-sources-4.13.5-r2.tbz2)...

Unfortunately, the developer container is looking in the wrong place. This is configured via `/etc/portage/make.conf` which has:

```
PORTAGE_BINHOST="http://builds.release.core-os.net/embargoed/devfiles/boards/amd64-usr/1520.6.0/pkgs/
http://builds.release.core-os.net/embargoed/devfiles/boards/amd64-usr/1520.6.0/toolchain/"
```

This one-line hack can be run in the development container to override that for this version.

```sh
source /etc/os-release && [ "${VERSION_ID}" == "1520.6.0" ] && export PORTAGE_BINHOST=$'http://builds.developer.core-os.net/boards/amd64-usr/1520.6.0/pkgs/\nhttp://builds.developer.core-os.net/boards/amd64-usr/1520.6.0/toolchain/'
```


This happened because this particular release and associated developer container were built under an embargo, and then when the embargo lifted we copied them over to the public buckets.

We'll fix the underlying problem so this sort of thing doesn't happen again (e.g. so for embargoed builds we should still set PORTAGE_BINHOST to the final upload location, as we already do for a few other artifacts).

We also expect to do a round of releases early next week which will have the correct dev image configuration, so if waiting for those seems like the best choice, those should fix this.

We might be able to duplicate the files back to that 'builds.release' location and make it accessible just for this version; I'll update this issue if we do decide to do that.

**Comment by euank** [comment]
After some discussion, we've decided to roll a new set of releases (one per channel) to fix this. We expect them to be rolling out tomorrow.

The root cause of this should also be addressed with https://github.com/coreos/scripts/pull/766 for the next such release.

**Comment by euank** [comment]
We've released 1520.7.0, 1548.3.0, and 1562.2.0 (in the stable, beta, and alpha channels respectively).

Those versions should have functioning developer containers. We're adding testing to ensure this doesn't happen again.

Thanks for reporting and apologies for the mess up on our side

**Comment by mumoshu** [comment]
@euank I appreciate your quick responses. I'll try to confirm if everything is ok asap 👍
